test/steps/utils.test.js

```diff
@@ -70,6 +70,11 @@ describe('Testing wrapNodes', () => {
 
 describe('Optimize Image URLs', () => {
   it('creates correct image optimize urls', () => {
+    assert.throws(() => optimizeImageURL(313), {
+      name: 'TypeError',
+      message: /^Parameter 'url' must be a string/,
+    });
```

Comment:
doesn't t work like this?

```suggestion
    assert.throws(() => optimizeImageURL(313), new TypeError("Parameter 'url' must be a string"));
```

Reply:
No, the error message dynamic and matches a substring.